README.md

```diff
@@ -11,24 +11,44 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be pulled locally from either dev namespace using the hash or production namespace.
+
+    Dev:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
```

Comment:
I would simply go over how to run it in dev mode, then at the end include any caveats for production, if any.

Reply:
That is totally true for the developer but for some one, who is just running the application, they don't need to run the dev version unless they want to run unreleased image.